### PR TITLE
Added tabs for my projects and all projects

### PR DESCRIPTION
**Pull request by GauravGusain98**
Targets #900 

### Description

We need to show list of projects in which user is assigned and the list of all projects in the project page. 

Here is the design of the page 
image

image

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/index.blade.php [hunk 1]
```diff
@@ -5,7 +5,20 @@
...
+                ({{ $projects ? $projects->count() : ''}})</h4>
+            <form class="mx-md-3" action="{{ route('project.index', ['status' => 'active'])  }}">
+                <input type="hidden" name="status" value="{{ request()->input('status', 'active') }}">
+                <select class="fz-14 fz-lg-16 p-1 bg-info text-white rounded border-0" name="projects"
+                    onchange="this.form.submit()">
+                    <option value="my-projects" {{ request()->get('projects') == 'all-projects' ? '' : 'selected' }}>
+                        {{ __('My Projects') }} </option>
+                    <option value="all-projects" {{ request()->get('projects') == 'all-projects' ? 'selected' : '' }}>
+                        {{ __('All Projects') }} </option>
+                </select>
```
Comment (GauravGusain98) [inline Modules/Project/Resources/views/index.blade.php:19]: Select box to choose between `my projects` and `all projects`

File: Modules/Project/Services/ProjectService.php [hunk 2]
```diff
@@ -11,8 +11,13 @@ class ProjectService implements ProjectServiceContract
 {
     public function index()
     {
-        return Project::where('status', request()->input('status', 'active'))
+        if (request()->get('projects') == 'all-projects') {
+            return Project::where('status', request()->input('status', 'active'))
+                ->get();
+        } else {
+            return auth()->user()->projects()->where('status', request()->input('status', 'active'))
             ->get();
```
Comment (GauravGusain98) [inline Modules/Project/Services/ProjectService.php:19]: returning the project list based on user selection.

File: Modules/User/Entities/User.php [hunk 3]
```diff
@@ -105,4 +106,9 @@ public function appointmentSlots()
     {
         return $this->hasMany(AppointmentSlot::class, 'user_id');
     }
+
+    public function projects()
+    {
+        return $this->belongsToMany(Project::class, 'project_resources', 'resource_id', 'project_id');
+    }
```
Comment (GauravGusain98) [inline Modules/User/Entities/User.php:113]: Relationship of projects and user

**Review by kuldeep3 — APPROVED** [review]
Looks good on code 👍🏽 . Left few comments.

File: Modules/Project/Resources/views/index.blade.php [hunk 4]
```diff
@@ -5,7 +5,20 @@
     @include('project::menu_header')
     <br>
     <div class="d-flex justify-content-between mb-2">
-        <h4 class="mb-1 pb-1">{{ config('project.status')[request()->input('status', 'active')] }} Projects ({{ $projects ? $projects->count() : ''}})</h4>
+       <span class="d-md-flex">
```
[the remark this answers is not in the document]
Comment (GauravGusain98) [reply Modules/Project/Resources/views/index.blade.php]: I made it similar to the `add project`. There span is used, but I got your point, it should be div as we we are using from inside a inline element

**Pull request merged**